src/envoy/http/jwt_auth/jwt.h

```diff
@@ -31,74 +32,83 @@ enum class Status {
   OK = 0,
 
   // JWT token is required.
-  JWT_MISSED,
+  JWT_MISSED = 1,
```

Comment:
not need to add =1,  
let compiler assign them.

In my PR,  I will remove BEARER_PREFIX_MISMATCH,  then I would have to change each assignment  too

Reply:
Per offline discussion, I use assignment here to make it easier for debugging. This enum class is updated.